engine/components/sql/src/main/scala/pl/touk/nussknacker/sql/db/ignite/TableCatalog.scala

```diff
@@ -0,0 +1,34 @@
+package pl.touk.nussknacker.sql.db.ignite
+
+import org.apache.flink.api.common.typeinfo.TypeInformation
+import pl.touk.nussknacker.engine.api.typed.typing.{Typed, TypedObjectTypingResult, TypingResult}
+
+import java.sql.Timestamp
+import java.time.LocalDateTime
+
+object TableCatalog {
+  case class ColumnMeta(name: String, private val klass: Class[_], isPartitionColumn: Option[Boolean]) {
+    val normalizedClass: Class[_] = {
```

Comment:
I wonder, maybe we want to transform Timestamp to some other class during execution? Is java.sql.Timestamp accessible from NU POV

Reply:
That's a good point, the mapping is even present in HsqlSqlQueryableDataBase I need to work out how to use it here...